Feature/fix/fragment family

AndreaPitossi opened the pull request.

AndreaPitossi committed:
Restyling del fragment family

AndreaPitossi committed:
Restyling del fragment family member

AndreaPitossi committed:
Restyling del fragment family member list

AndreaPitossi committed:
Restyling dell'activity family member detail

AndreaPitossi committed:
Restyling dei fragment achievement e achievement list

AndreaPitossi committed:
Restyling della lista degli achievement con aggiunta del titolo nell'activity dettagli family member

AndreaPitossi committed:
Sistemato il passaggio alla detail activity del membro

AndreaPitossi committed:
Sistemato warning

AndreaPitossi committed:
VIsualizzato ruolo nella lista dei membri della famiglia

AndreaPitossi committed:
Messa icona della matita nella toolbar del dettaglio del membro

MarcoCadei commented:
Ok, ottimo mi piace tutto meno due cose un po' strane:
1. I punti nel dettaglio del membro mi sembrano allineati a caso? CIoè non mi ricordo se è così che lo volevamo, ma io li metterei o a destra (come nella lista), oppure allineati a sinistra sotto il nome (sono più propenso per questo
2. Ora che lo vedo il bottone elimina non mi convince molto! Elimina e edit sono sempre vicini, quindi perché non lo mettiamo nella toolbar?

AndreaPitossi commented:
1 - Neanche a me puaceva cosi... ma mi sembrava fosse cosi che avevamo detto... attendo il parere di GR per la modifica
2 - E fare entrambi come bottoni? Non lo so eh... in ogni caso attendiamo il parwre di GR... poi provvedo

MarcoCadei commented:
2 - Sìsì, anche. La mia proposta è solo per cambiare la situazione corrente, se si trovano proposte migliori ben vengano

gianluca150 commented:
1 - Sì, mi sembra di ricordare anch'io che avevamo detto di farla diversa in questa schermata l'indicazione dei punti dato che c'è l'immagine a sinistra e altrimenti ci potrebbero essere problemi con il nome, detto questo però se non vi piace la possiamo benissimo cambiare come più vi pare e piace.
2 - Per me va bene metterlo nella toolbar. Tecnicamente comunque per com'è ora l'applicazione la matita e il cestino si vedono assieme solo se l'utente che ha fatto l'accesso è l'owner che è l'unico in grado di modificare ed eliminare tutti quanti (e tra l'altro ora che ci penso l'owner non dovrebbe poter eliminare sè stesso perché sennò la casa rimane senza owner; aggiungo tra le minuzie un "mettere la possibilità di cambiare owner" ma per adesso non ce ne curiamo). Detto ciò comunque come soluzione mi va bene anche quella di spostare il modifica dalla toolbar ad un altro bottone in mezzo alla schermata... forse così è un po' più consistente con il resto ma effettivamente quello è l'unico punto dove c'è una modifica vera e propria, quindi se lo lasciamo diverso ha comunque senso

gianluca150 committed:
Modificato lo stile della FamilyMemberDetailActivity
Aggiunto nested scrolling per la lista di achievement posizionata sotto i dettagli del membro della famiglia

gianluca150 committed:
Revert commit d4604da80eb235698a01dfbfa4a6ca7adde7256b
Rimosso menù con l'icona di modifica dalla toolbar della FamilyMemberDetailActivity

gianluca150 reviewed (approved):
Modifiche effettuate secondo quanto detto oggi

AndreaPitossi commented:
Togliere doppia scrollview dal verticale e creare un differente layout in orizzontale
